lib/i18n/app_de.json

```diff
@@ -42,6 +42,10 @@
     "applyFilter": "Filter anwenden"
   },
   "campaigns": {
+    "debug": {
+      "confirm_storing_cache": "Falls Du Probleme beim Synchronisieren hast, kannst Du die lokalen Änderungen nun auf Deinem Gerät abspeichern. Willst Du das?",
```

Comment:
Still think this would make me slightly panicky because DO I WANT THIS but of course up to @JuliaFreitagBGSt

Reply:
It's a non-official functionality and it's confirmed by the BGSt. ;)